src/components/shared/Rating/index.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { useState } from 'react';
+
+import styled from 'styled-components/native';
+
+const COLOR: {
+  [key: string]: string;
+} = {
+  WHITE: '#ffffff',
+  BACKGROUND: '#EFF0F1',
+  ACTIVE: '#FFB402',
+  INACTIVE: '#BDBDBD',
+};
+interface Props {
+  total: number;
+  value: number;
+  onChange: Function;
```

Comment:
`onChange: () => void` is normal when you want to use function type definition with no argument.

Reply:
Ph thanks!